Gemfile

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem 'relaton-cli'
```

Comment:
It doesn't need the relaton-cli when we use direct call without `system` CLI call.